### PR TITLE
[LC-625] Timer(Service) unit tests

### PR DESCRIPTION
**Pull request by metalg0su**
# Goal
- More precise test
> Test for callbacks both blocking and coroutine func.
- Faster Unit test for `Timer` & `TimerService`
> 21+ sec -> 2.0- sec
- Increase coverage for `Timer` & `TimerService`
> 79% -> 96%+

### CODE REVIEW

**Comment by metalg0su** [comment]
retest this please

File: loopchain/baseservice/timer_service.py [hunk 1]
```diff
@@ -66,9 +66,13 @@ def is_timeout(self) -> bool:
 
     @property
     def is_repeat(self) -> bool:
-        if self._is_repeat and \
-                (self._repeat_timeout == 0 or
-                 (time.time() - self.__repeat_start_time < self._repeat_timeout)):
+        is_repeatable = self._is_repeat
+        if not is_repeatable:
```
Comment (windies21) [inline loopchain/baseservice/timer_service.py]: This temp variable is needed?

**Review by windies21 — APPROVED** [review]

File: loopchain/baseservice/timer_service.py (hunk 1 above)
Comment (metalg0su) [reply loopchain/baseservice/timer_service.py]: Thanks. You're right.

Comment (metalg0su) [reply loopchain/baseservice/timer_service.py]: I removed 
FROM:
```
is_repeatable = self._is_repeat
if not is_repeatable:
```

TO:
```
+        if not self._is_repeat:
+            return False
```

**Pull request merged**